src/components/Comments.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from "react"
+import { getComments } from "../api"
+
+const Comments = ({article_id}) => {
+
+
+    const[comments, setComments] = useState([]);
+    const[isLoading, setIsLoading] = useState(true);
+    const [isError, setIsError] = useState(null);
+  
+
+
+    useEffect(() => {
+
+        getComments(article_id).then((commentsObject) => {
```

Comment:
This useEffect could be re-triggered, in which case we would not set loading state to be true. I would prefer it if we also set loading to be true right at the top of this useEffect()